scripts/simple_app_benchmark_upload.py

```diff
@@ -29,20 +29,21 @@ def extract_stats_from_json(json_file: str) -> list[dict]:
 
     # Iterate over each test in the 'benchmarks' list
     for test in data.get("benchmarks", []):
+        group = test.get("group", None)
         stats = test.get("stats", {})
+        full_name = test.get("fullname")
+        file_name = (
+            full_name.split("/")[-1].split("::")[0].strip(".py") if full_name else None
+        )
         test_name = test.get("name", "Unknown Test")
-        min_value = stats.get("min", None)
-        max_value = stats.get("max", None)
-        mean_value = stats.get("mean", None)
-        stdev_value = stats.get("stddev", None)
 
         test_stats.append(
             {
                 "test_name": test_name,
-                "min": min_value,
-                "max": max_value,
-                "mean": mean_value,
-                "stdev": stdev_value,
+                "group": group,
+                "stats": stats,
+                "full_name": full_name,
```

Comment:
what is the full_name here?

Reply:
full_name is `<test_file_name>::<testname>`